File: test_req.js
const fs = require('fs');
const env = fs.readFileSync('.env', 'utf8');
const token = env.split('\n').find(l => l.startsWith('VITE_TEMP_TOKEN=')).split('=')[1].replace(/["']/g, '').trim();

fetch('http://localhost:8000/api/v1/connections/e1c1d7bd-77ab-41dc-bc0d-dd4d06cec80b/test', {
  method: 'POST',
  headers: {
    'Authorization': 'Bearer ' + token
  }
}).then(r => r.text().then(t => console.log(r.status, t)));

File: test_req.py
import os
import requests
import json

def test():
    token = None
    with open('.env', 'r') as f:
        for line in f:
            if line.startswith('VITE_TEMP_TOKEN='):
                token = line.split('=', 1)[1].strip().strip('"').strip("'")
                break
    
    if not token:
        print("No token found")
        return

    url = 'http://localhost:8000/api/v1/connections/e1c1d7bd-77ab-41dc-bc0d-dd4d06cec80b/test'
    headers = {'Authorization': f'Bearer {token}'}
    res = requests.post(url, headers=headers)
    print("Status:", res.status_code)
    print("Response:", res.text)

if __name__ == '__main__':
    test()
